src/cli.rs

```diff
@@ -36,9 +36,19 @@ pub fn get_cli() -> App<'static, 'static> {
                 .help("Output to selection (defaults to clipboard)"),
         )
         .arg(
-            Arg::with_name("no_preview")
```

Comment:
This is a breaking change, since the `--no-preview` flag has been removed (along with its accompanying environment variable).

This is because we no longer use the `SHAPE` X extension to draw a circular window (see below comments for how it's drawn).